exasol/toolbox/tools/replace_version.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+from typing import List
+
+def replace_version(template: Path, version: str) -> None:
+    with open(template, "r+", encoding="utf8") as file:
+        lines = file.readlines()
+        _replace(lines, "exasol/python-toolbox/.github/", version)
+        file.seek(0)
+        file.writelines(lines)
```

Comment:
I'll suggest you do IPO (Input Processing Output) / EVA (Eingabe Verarbeitung Ausgabe).

1. Read the file/lines (open read)
2. Transform data
3.  Write new data (open write)